addons/survey/models/survey_survey.py

```diff
@@ -678,6 +678,44 @@ def _get_conditional_maps(self):
                     triggered_questions_by_answer[question.triggering_answer_id] = question
         return triggering_answer_by_question, triggered_questions_by_answer
 
+    # ------------------------------------------------------------
+    # BACKGROUND MANAGEMENT
```

Comment:
HumanLinter: would put it in QUESTIONS MANAGEMENT section (still a question-related tool method)